lab-raziyeh/app/config/router-config.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = ['$stateProvider', '$urlRouterProvider',routerConfig ];
+function routerConfig ($stateProvider, $urlRouterProvider){
+  $stateProvider
```

Comment:
@Raziyehbazargan i'd have this touching the following line, for readability - this almost looks like a global var declaration because of the extra line and indentation